doc/whats-new.md

```diff
@@ -1,200 +1,476 @@
-(i.e. ruptures within the integration distance) even for classical
-calculations.
+The `ucerf_rupture` calculator has been extended so that we can
+parallelize by number of stochastic event sets. This made it possible
+to run mean field calculations in parallel: before such calculations
+used a single core, since there is a single branch.
 
-The `openquake.cfg` file is included in the installation, so users that
-installed the engine via `pip install openquake.engine` will not get
-any complain about the missing configuration file.
+The data transfer has been hugely reduced in the calculator
+`ucerf_risk`: now we do not return the rupture objects from the
+workers, but only the event arrays, which are enough for the purposes
+of the calculator. This saved around 100 GB of data transfer in large
+calculations for California.
 
-The repository with the latex manual has been merged into the oq-engine
-repository, to help reducing disalignement between the code and the
-documentation.
+We fixed a bug in `ucerf_risk` that prevented the average losses
+from being stored. Now this works out of the box, provided you
+set `avg_losses=true` in the `job.ini` file.
+
+There is a brand new time-dependent UCERF classical calculator.
 
-We fixed an issue with rabbitmq on Ubuntu 16.04 by updating our configuration
-file `openquake.cfg` to not use the `guest` account.
+We started working on an event based calculator starting from Ground
+Motion Fields provided by the user. The current version is still very
+preliminary and requires the GMFs to be in NRML format, but we plan
+to extend it to read the data from more efficient formats (CSV, HDF5)
+in the near future.
+
+We added a facility to serialize `Node` objects into HDF5 files. This is
+the base for a future development that will allow to serialize point sources
+into HDF5 datasets efficiently (scheduled for engine 2.5).
+
+We introduced some preliminary support for the Grid Engine. This is useful
+for people running the engine on big clusters and supercomputers. Unfortunately
+since we do not have a supercomputer we are not able to really test this 
+feature. Interested users should contact us and offer some help, like giving
+us access to a Grid Engine cluster.
+
+Travis/Jenkins/packaging (check these with Daniele)
+---------------------------------------------------
+
+Fix an issue with Travis branches when submitting PR
+Change how `local_settings.py` is found
+Make init scripts more robust and portable
+Turned matplolib into a required library and added a plotting test on Travis
+Check if oq is talking to a foreign DbServer
+Sync packager.sh with oq-hazardlib to manage oq-libs-extra
+Bump to h5py 2.7.0
+Differentiate listening interface and connect interface in dbserver
+Small fixes for local_settings template to be able to append settings
+Add a template for PAM auth
+Split RPM packages in standard, master, worker
+Use CDN ftp mirror instead of upstream
```

Comment:
I would keep only:

Change how `local_settings.py` is found
I need to explain the logic

Turned matplolib into a required library and added to our distributions

 Check if oq is talking to a foreign DbServer

Bump to h5py 2.7.0

Add a template for PAM auth

Split RPM packages in standard, master, worker

Everything else isn't for users and may be only bring more confusion